src/lib/viewers/image/ImageViewer.js

```diff
@@ -81,6 +81,11 @@ class ImageViewer extends ImageBaseViewer {
 
         this.currentRotationAngle = 0;
 
+        const fileName = this.options.file && this.options.file.name && this.options.file.name.split('.')[0];
+        if (fileName) {
+            this.imageEl.setAttribute('alt', `preview image ${fileName}`);
```

Comment:
We'll need to make sure we have internationalized strings for this text, should we want to use it. It seems like passing the file name on its own to the alt attribute would be adequate, though.